test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -159,12 +169,54 @@ public void CreateBindingMetadata_EditableAttributeTrue_SetsReadOnlyFalse()
             Assert.False(context.BindingMetadata.IsReadOnly);
         }
 
+        [Fact]
+        public void CreateDisplayMetadata_DisplayAttribute_NameFromResources_UsesDataAnnotationLocalizaterProvider()
+        {
+            // Arrange
+            var sharedLocalizer = new Mock<IStringLocalizer>(MockBehavior.Loose);
+            sharedLocalizer
+                .Setup(s => s["DisplayName"])
+                .Returns(() => null);
```

Comment:
minor: No need to set anything up when using `MockBehavior.Loose`. Return value will be `null` anyhow.

But more generally, isn't this an impossible scenario? When would a real `IStringLocalizer` return `null`?